Source/Backend/HouseholdManager.Api/Domain/ShoppingListItem.cs

```diff
@@ -28,8 +30,15 @@ public void SetAmount(int amount)
         Amount = amount.Throw().IfLessThan(Conventions.MIN_AMOUNT);
     }
 
+    public void SetProductInfo(ProductInfo productInfo)
+    {
+        ProductInfo = productInfo.ThrowIfNull();
```

Comment:
Product info sollte noch selbst validieren

Reply:
Willst du den Namen validieren oder what?

Reply:
Not empty und den ganzen kram sollte ProductInfo auch mitmachen. Es hilft ja nix wenn das leer bleibt.